src/services/items/interfaces/item.ts

```diff
@@ -1,18 +1,21 @@
+import { ItemType } from '../../../constants/itemType';
 import { Serializable, UnknownExtra } from '../../../interfaces/extra';
 
-export interface Item<T extends UnknownExtra = UnknownExtra> {
+export interface ItemSettings extends Serializable {
+  isPinned?: boolean;
+  showChatBox?: boolean;
+  hasThumbnail?: boolean;
+}
+
+export interface Item<T extends UnknownExtra = UnknownExtra, S = ItemSettings> {
```

Comment:
Out of curiosity why was the settings type parameterized?

Reply:
For it to work in query-client, since it was playing with Record..